### PR TITLE
docs: describe how to implement CCs

### PR DESCRIPTION
**Pull request by AlCalzone**

### CODE REVIEW

File: docs/development/implementing-cc.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Implementing a Command Class
+
+Make sure you have the **project snippets** extension for VSCode installed. Implementing a Command Class (CC) requires **a lot** of boilerplate code and you won't have fun without that extension.
```
Comment (robertsLando) [inline docs/development/implementing-cc.md]:
```suggestion
Make sure you have the [project snippets](https://marketplace.visualstudio.com/items?itemName=rebornix.project-snippets) extension for VSCode installed. Implementing a Command Class (CC) requires **a lot** of boilerplate code and you won't have fun without that extension.
```

**Review by robertsLando — APPROVED** [review]
LGTM. Did you take a look to the snippets? You said them are a little bit old and maybe some needs updates? :)

**Comment by AlCalzone** [comment]
The CC ones should be up to date - I use them regularly. There are some others (probably outdated ones) for message classes and tests, but I don't mention them here ;)

**Comment by robertsLando** [comment]
@AlCalzone Ok perfect so :+1:

**Pull request merged**